certbot-apache/certbot_apache/configurator.py

```diff
@@ -573,7 +574,7 @@ def get_virtual_hosts(self):
                 ("/files%s//*[label()=~regexp('%s')]" %
                     (vhost_path, parser.case_i("VirtualHost"))))
             paths = [path for path in paths if
-                     os.path.basename(path) == "VirtualHost"]
+                     "VirtualHost" in os.path.basename(path)]
```

Comment:
We need something like `"virtualhost" in os.path.basename(path).lower()` here to catch lowercase statements.